tests/unit_tests/pyanaconda_tests/ui/test_webui.py

```diff
@@ -74,6 +74,7 @@ def test_error_propagation(self, mocked_print_message):
 
         mocked_print_message.reset_mock()
         self.intf.showDetailedError("My detailed error", "Such a detail!")
+        # pylint: disable=implicit-str-concat
```

Comment:
Honestly, this seems to me like a bug. If I'm looking on the string correctly than it should be starting with `"""` instead of `"`.

Reply:
Yes! missed it.